client/push.ts

```diff
@@ -1,12 +1,28 @@
 import {html, render} from './node_modules/lit-html/lit-html.js';
 
+declare var PushManager: {
```

Comment:
This is actually not augmenting the global `PushManager` variable. Since this is an ES module, the `var` is not globally scoped, it's scoped to the module, so you're declaring that there will be a different variable called `PushManager`. I guess it will actually work, because no JS will be emitted for this `declare` statement, so when you reference it below you'll end up referencing the global symbol anyway, but it feels kind of wrong to trick the compiler this way.

To add a declaration to the global scope from a module, you have to do it explicitly like this:

```
declare global {
  var foo ...
}
```

However, this won't work for us here because you actually just can't do declaration merging with variables (if you try it, it will say you can't re-declare `PushManager` with a different type).

IMO the TypeScript declarations seem wrong. `var PushManager` is actually an instance with type `interface PushManager`, but they way they are written (https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L9142) suggests that `var PushManager` is a constructor that returns a new `interface PushManager`. Maybe we should submit a PR to fix it (as well as adding `supportedContentEncodings`).

Sadly, for now I think you should just do `(PushManager as any).supportedContentEncodings` and add a comment.